site/tests/model_validation/ragas/ContextUtilization.md

```diff
@@ -1,87 +0,0 @@
-# ContextUtilization
```

Comment:
Agreed, but I am not sure I am the one who can help solve this scenario. 😆 I assume you are just thinking out loud as a reminder to maybe bring this up with product management? (Since we really only get notified when we retrieve the source and find out something was removed...)